libs/api/domains/cms/src/lib/models/organizationPage.model.ts

```diff
@@ -35,6 +36,9 @@ export class OrganizationPage {
 
   @Field(() => [FooterItem], { nullable: true })
   footerItems?: Array<FooterItem>
+
+  @Field(() => [SidebarCard], { nullable: true })
+  sidebarCards?: Array<SidebarCard>
```

Comment:
Seems like these always return array, maybe the nullable and optional are not needed?